src/users/users.routing.ts

```diff
@@ -30,6 +30,11 @@ export const ROUTES: Routes = [
     /*{
       path: 'user/:userId/editpermission',
       component: EditUserPermissionComponent,
+    }
+    {
```

Comment:
Should we not remove comments? Are these lines going to be reactivated later?

Reply:
Reactivated later.